docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  database:
+    image: 'postgres:latest'
+    container_name: 'fit_database'
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: fit_tracker
+    volumes:
```

Comment:
We don't need it, the data will be persisted in the container, let's get rid of that.

Reply:
After deleting, cannot register server in pgAdmin4. For blank username and password:
'''
Username must be specified
'''
With username: user and any password
'''
Unable to connect to server: connection failed: FATAL: role "user" does not exist
'''

Reply:
As discussed, only `volumes` had to be removed. It's ok now, so resolving.